_posts/2021-05-12-244.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Cockpit 244
+author: kkoukiou
+date: 2021-05-12
+tags: cockpit linux
+slug: cockpit-244
+category: release
+summary: Cockpit with various improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 244 and Cockpit-Machines 244.
+
+## Shell: No opportunistic sudo invocation
+
+Cockpit will no longer try to run sudo on the very first login, and you have to
+explicitly turn on administrative access once for each browser profile that you
+use to access Cockpit. This prevents cockpit's use of sudo being noisy in the
+journal.
+
+![Sudo error messages](/images/sudo-error-messages.png)
+
+## Machines: Edit the MAC address of a VM's network interface
+
+To edit the MAC address of a virtual machine click the `Edit` button on the row
```

Comment:
nit: I wonder if backticks are the right markup option here?  Is there a better tag for "UI element" that doesn't imply "fixed width"?

Reply:
No idea, to be honest we never did guide the user int the UI in this way in the past in a blog post. I was not even sure if we want this kind of Ui guidance here.

Reply:
Ya.  I agree.  it's oddly specific.  Maybe take it out?

Reply:
It could be "Edit", as we're quoting the button's text. This is pretty standard in software documentation.

```suggestion
To edit the MAC address of a virtual machine click the "Edit" button on the row
```

There's also the `<kbd>` tag, but that's more for keystrokes, not UI elements. And it also makes the text fixed-width.

Reply:
I have no problem @allisonkarlitskaya, I just added as a filler to be honest. @garrett shall I take it out?